src/config/section/event.js

```diff
@@ -44,7 +44,9 @@ export default {
         ids: {
           value: (record) => { return record.id }
         }
-      }
+      },
+      show: (record, store) => { return !['User'].includes(store.userInfo.roletype) },
+      groupShow: (record, store) => { return !['User'].includes(store.userInfo.roletype) }
```

Comment:
@davidjumani why not use the show as the implementation is same - the condition for showing the button can remain same? (or does it conflict with other views like stop/start vms etc.)?

Reply:
Yup. Causes issues and shows stop / start based on the first vm's state